### DISCUSSION TITLE
Multiple Pseudo-Classes

### DISCUSSION THREAD

**Discussion by brunodavi** [tailwindlabs/tailwindcss#12436] [Help]
Hello Tailwind community! I'm exploring the possibility of using multiple classes within a `hover` state to reduce repetition. For instance, rather than the conventional:

```html
<p className="hover:text-white hover:bg-black">Hover Text</p>
```

Is there an equivalent shorthand like:

```html
<p className="hover:[bg-black text-white]">Hover Text</p>
```

I came across a Twitter discussion (link: https://twitter.com/adamwathan/status/1461519820411789314) that introduced a proposal for a more concise hover syntax:

```html
<p className="hover:(text-white,bg-black)">Hover Text</p>
```

While this appears to be an intriguing idea, I haven't found any related discussions on GitHub or confirmation of its implementation. Have you encountered any information on this or similar proposals?

Additionally, are there any best practices to avoid unnecessary repetition of pseudo-classes? Your insights would be greatly appreciated!

**Comment by wongjn** [comment]
There have been suggestions about adding such a feature. The maintainers have addressed this request before, as per the full thread of tweets from the tweet you mentioned. There is also [a previous GitHub discussion thread with maintainer responses](https://github.com/tailwindlabs/tailwindcss/discussions/8337#discussioncomment-4032611).

**Reply by brunodavi** [reply]
Is there an official way to use it in the documentation or do you know and have tested it, as I remember testing with `hover:(text-black,bg-black)` but with no effect and with square brackets or changing the comma for a space

**Reply by wongjn** [reply]
No, there is no public release of this feature.

**Reply by brunodavi** [reply]
Right, thanks
